custom/icds_reports/urls.py

```diff
@@ -121,7 +121,8 @@
     url(r'^bug_report/', ICDSBugReportView.as_view(), name='icds_bug_report'),
     url(r'^awc_locations/', AWCLocationView.as_view(), name='awc_locations'),
     url(r'^download_pdf/', DownloadPDFReport.as_view(), name='icds_download_pdf'),
-    url(r'^issnip_pdf_status/', CheckPDFReportStatus.as_view(), name='issnip_pdf_status'),
```

Comment:
Could we keep this URL for now, and then remove it in a follow up PR? That way links that have already been generated can continue working. I think it should be able to point to `CheckExportReportStatus`